src/plugins/intel_gpu/include/intel_gpu/primitives/gather.hpp

```diff
@@ -41,6 +41,42 @@ struct gather : public primitive_base<gather> {
         , batch_dim(batch_dim)
         , support_neg_ind(support_neg_ind) {}
 
+    /// @brief Constructs gather compressed primitive.
+    /// @param id This primitive id.
+    /// @param dict Input dictionary primitive id.
+    /// @param idx Input indexes primitive id.
+    /// @param axis Gathering axis.
+    /// @param compression_scale Primitive id containing scale factors for weights decompression.
+    /// @param compression_zero_point Primitive id containing zero points for weights decompression.
+    /// @param input_rank Input rank.
+    /// @param output_shape Output shape.
+    /// @param batch_dim Batch_dim
+    /// @param support_neg_ind Support negative indexes
+    gather(const primitive_id& id,
+           const input_info& dict,
+           const input_info& idx,
+           const int64_t axis,
+           const primitive_id& decompression_scale,
+           const primitive_id& decompression_zero_point,
```

Comment:
`primitive_id` -> `input_info`

Reply:
I updated it to use `input_info` instead of `primitive_id`.